garage/replay_buffer/simple_replay_buffer.py

```diff
@@ -25,16 +25,16 @@ def sample(self, batch_size):
             buffer[key] = self._buffer[key][:self._current_size]
```

Comment:
if `sample` changes the behavior from the base class, please specify in the docstring how it differs.